src/main/webapp/dataverse-textMessages.xhtml

```diff
@@ -106,7 +106,7 @@
                                     <ui:repeat value="#{banner.dataverseLocalizedBanner}" var="localbanner">
                                         <div class="p-col">
                                             <h:outputLabel for="banner-thumbnail"
-                                                           value="#{settingsWrapper.getConfiguredLocaleName(localbanner.locale)}:"
+                                                           value="#{bundle[localbanner.locale]}:"
```

Comment:
This won't work like an original solution - that is returning language translated to language of current localized banner: "polski", "english" etc.